### PR TITLE
Add Gamecontroller.php, game.css, folder Game and it view game.html.twig

### PR DESCRIPTION
**Pull request by StephaneSauvaget**

### CODE REVIEW

**Comment by Orijma** [comment]
Different indentation than my home.html.twig
Don't know which one Romain prefer
we'll see this soon

**Review by Orijma — APPROVED** [review]

File: src/Controller/GameController.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: aurelwcs
+ * Date: 08/04/19
+ * Time: 18:40
+ */
```
Comment (RomainClair) [inline src/Controller/GameController.php:8]: Seems false. Beware of copy/paste =)

File: src/View/Game/game.html.twig [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{% extends 'layout.html.twig' %}
+
+    {% block title %}
+        Stercket Battle
+    {% endblock %}
+
+    {% block link %}
+            <link rel="stylesheet" href="/assets/css/game.css">
+            <link rel="preconnect" href="https://fonts.gstatic.com">
+            <link href="https://fonts.googleapis.com/css2?family=Passero+One&display=swap" rel="stylesheet">
```
Comment (RomainClair) [inline src/View/Game/game.html.twig:10]: It's probably better to have self hosted fonts

File: src/View/Game/game.html.twig [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                <button class = "button buttonBattle"> Battle </button>
+            {% endblock %}
+        {% endblock %}
+        {% block collection %}
+                <button class = "buttonCollection"> Collection </button>
+        {% endblock %}
+        {% block stercket %}
+                <img src="" alt="">
+                <img src="" alt="">
+                <img src="" alt="">
```
Comment (RomainClair) [inline src/View/Game/game.html.twig:26]: Seems useless... and probably not valid

File: src/View/Game/game.html.twig [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            <link rel="stylesheet" href="/assets/css/game.css">
+            <link rel="preconnect" href="https://fonts.gstatic.com">
+            <link href="https://fonts.googleapis.com/css2?family=Passero+One&display=swap" rel="stylesheet">
+    {% endblock %}
+    {% block content %}
+        {% block nav %}
+            {% block button %}
+                <button class = "button buttonHome"> Home </button>
+                <button class = "button buttonRest"> Rest </button>
+                <button class = "button buttonBattle"> Battle </button>
```
Comment (RomainClair) [inline src/View/Game/game.html.twig:17]: Do you plan to have JS involved ?
If not this should probably be links or forms ot have a client/server interaction
And use CSS if you want them to looks like buttons

**Review by RomainClair — CHANGES_REQUESTED** [review]
You probably should name you branches and PR according to what you want to do, and not a list of files.
Probably something like : the main game page - first draft

**Pull request closed without merging**